CMakeLists.txt

```diff
@@ -206,6 +228,7 @@ if(WITH_SPDK)
   set(HAVE_SPDK TRUE)
 endif(WITH_SPDK)
 
+option(WITH_BLKID "Block device library" ON)
```

Comment:
so this is more about the platform limitation, maybe we can just disable it, like what you already do.

if we follow this logic further, we need to have a knob for every difference on platforms. this does not make sense to me at all.